src/gl.rs

```diff
@@ -754,6 +754,8 @@ declare_gl_apis! {
         data: *const GLvoid,
         flags: GLbitfield,
     );
+
+    fn flush_mapped_buffer_range(&self, target: GLenum, offset: GLintptr, length: GLsizeiptr);
```

Comment:
I looked at Servo to see if they still have a custom implementation of the trait. Looks like they no longer do, so this would be a non-breaking change in practice (but breaking in theory).